packages/edit-post/src/hooks/use-navigate-to-entity-record.js

```diff
@@ -34,32 +38,41 @@ export default function useNavigateToEntityRecord(
 			}
 			return historyState;
 		},
-		[ { postId: initialPostId, postType: initialPostType } ]
+		[
+			{
+				post: { postId: initialPostId, postType: initialPostType },
+			},
+		]
 	);
 
-	const initialPost = useMemo( () => {
-		return {
-			type: initialPostType,
-			id: initialPostId,
-		};
-	}, [ initialPostType, initialPostId ] );
```

Comment:
Why is `initialPost` removed from here and moved further down into the `Editor` component? (I'm just curious, this is not a request to make any changes here 🙂)

Reply:
It doesn't have anything to do with the primary purpose of the hook (building `onNavigateToEntityRecord` callbacks) so I split it out. Feels weird to me that you'd pass `initialPostType` and `initialPostId` to a hook and it returns exactly what you gave it in an object without doing anything else.

Reply:
Ah, gotcha. That makes sense, thanks!